### PR TITLE
change document title

### LINKED ISSUE #64: Remove the "reliable reset" phrasing from the draft
Resets are always reliable in QUIC itself, so the phrasing is in general confusing.  We already renamed the frame to RESET_STREAM_AT, so the rest of the draft should probably be updated too.

### PR DESCRIPTION
**Pull request by marten-seemann**
Fixes #64.

@kazuho You wanted to add some more text regarding the different types of partial reliability. Want to make a suggestion here?

### CODE REVIEW

**Comment by kazuho** [comment]
Merged main to incorporate other changes.

File: draft-ietf-quic-reliable-stream-reset.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
 ---
-title: "Reliable QUIC Stream Resets"
-abbrev: "QUIC Reliable Stream Reset"
+title: "QUIC Stream Resets with Partially Reliable Stream Data Delivery"
```
Comment (kazuho) [inline draft-ietf-quic-reliable-stream-reset.md:2]:
```suggestion
title: "QUIC Stream Resets with Partially Reliable Delivery"
```

To me it seems "stream data" is verbose (what else would you deliver on streams)? Maybe "reliable" also is.

**Review by kazuho — COMMENTED** [review]
Merged main to suggest changes inline but it seems I cannot.

@marten-seemann 
> You wanted to add some more text regarding the different types of partial reliability. Want to make a suggestion here?

Right, maybe add something like below (the sentence in bold) to the last paragraph of Introductions?

> This document extends QUIC with a variant of stream resets that reliably delivers the beginning of a stream up to a sender-specified offset, communicated using the RESET_STREAM_AT frame. **It can be considered a form of partial reliability that is range-based.** As a variant of reset, application protocols continue to treat this stream function as an abrupt termination; see [Section 2.4](https://rfc-editor.org/rfc/rfc9000#section-2.4) of [[RFC9000](https://quicwg.org/reliable-stream-reset/draft-ietf-quic-reliable-stream-reset.html#RFC9000)].

**Comment by marten-seemann** [comment]
Rebased and applied both of @kazuho's suggestions.

**Review by kazuho — APPROVED** [review]

**Pull request merged**